CONTRIBUTING.md

```diff
@@ -36,11 +36,11 @@ Here comes a little surprise: You need [Node.JS](http://nodejs.org).
 
 ### 2. Install the dependencies
 
-Just "cd" into sequelize directory and run `npm install`, see an example below:
+Just "cd" into sequelize directory and run `npm ci`, see an example below:
```

Comment:
`npm ci` is meant to be used in CI environments, right? I think we should not ask contributors to use this on their machine. Remove changes from this file

Reply:
No, https://docs.npmjs.com/cli/ci.html
`ci` ensures it installs exactly what is in the package-lock.json file, `install` will only look at the package.json

Reply:
That being said, while not *required* for developers it's always a good idea to ensure the dep tree is the same as in the CI to avoid confusion.

Reply:
Not sure if I still prefer this but no problem with this change